shub/image/list.py

```diff
@@ -52,31 +51,17 @@ def list_cmd_full(target, silent, version):
     image = config.get_image(target)
     version = version or config.get_version()
     image_name = utils.format_image_name(image, version)
```

Comment:
I reverted some changes, now `config.get_image` raises different exceptions whether image is not found or image is False for a target, let me know if you disagree. The `get_image` function is used only by `shub.image` commands, and in my mind it should fail if image is expected but missing; otherwise we could add another `get_image_or_fail` with these exceptions.